deploy/examples/nfs-test.yaml

```diff
@@ -4,12 +4,9 @@ metadata:
   name: my-nfs
   namespace: rook-ceph # namespace:cluster
 spec:
-  # rados settings aren't necessary in Ceph Versions equal to or greater than Pacific 16.2.7
   rados:
```

Comment:
cluster-test.yaml uses 16.2.7 now, shall we still leave a comment here about `rados` being ignored in that case?